server/odc-service/src/main/java/com/oceanbase/odc/service/task/TaskService.java

```diff
@@ -287,6 +287,15 @@ public void succeed(Long id, Object taskResult) {
         log.info("Task ended: taskId={}", id);
     }
 
+    @Transactional(rollbackFor = Exception.class)
```

Comment:
we already have ` public void succeed(Long id, Object taskResult)` , seems not necessary add a method with less responsibility. may handle `null` taskResult inside current `succeed` method.

so as method ` public void succeed(Long id, Object taskResult)`.

Reply:
ok